client/src/components/Table.js

```diff
@@ -0,0 +1,108 @@
+import React, {Component} from "react";
+import "../css/Table.css";
+import "../css/themePalette.css";
+import { Table } from "react-bootstrap";
+
+
+
+// The Table
+class LogTable extends Component {
+  constructor() {
+    super();
+    // Made dummy data as state becuase it might be easier to change later
+    this.state = {
```

Comment:
Component state should only be used for data that is changed by the user, just use a constant here.